### PR TITLE
Edit

### PR DESCRIPTION
**Pull request by Megha-Joshi**
In this PR, I have added edit note feature.

### CODE REVIEW

File: src/note-components/Homepage/homepage.jsx [hunk 1]
```diff
@@ -64,6 +73,19 @@ console.log(error);
...
 
+const customStyle = {
+overlay: {
+top: "6rem",
+backgroundColor: "rgba(52, 58, 64, 0.8)",
+},
+content: {
+width: "18rem",
+// height: "20rem",
+margin: "5rem auto",
```
Comment (srishtigautam21) [inline src/note-components/Homepage/homepage.jsx:84]: remove commented code

File: src/note-components/Homepage/homepage.css [hunk 2]
```diff
@@ -213,4 +213,73 @@
     display: flex;
     justify-content: center;
     align-items: center;
+}
+
+/*  */
+
```
Comment (srishtigautam21) [inline src/note-components/Homepage/homepage.css:219]: remove this

**Review by srishtigautam21 — APPROVED** [review]
The code looks good to merge

File: src/note-components/Homepage/homepage.jsx [hunk 3]
```diff
@@ -64,6 +73,19 @@ console.log(error);
...
 }
 
+const customStyle = {
+overlay: {
+top: "6rem",
+backgroundColor: "rgba(52, 58, 64, 0.8)",
+},
+content: {
+width: "18rem",
+// height: "20rem",
```
Comment (zoyatanweer) [inline src/note-components/Homepage/homepage.jsx:83]: remove commented code

**Review by zoyatanweer — APPROVED** [review]
everything looks good to me. Good to merge

File: src/note-components/Homepage/homepage.jsx (hunk 1 above)
Comment (Megha-Joshi) [reply src/note-components/Homepage/homepage.jsx:84]: Okay

File: src/note-components/Homepage/homepage.css (hunk 2 above)
Comment (Megha-Joshi) [reply src/note-components/Homepage/homepage.css:219]: Okay

File: src/note-components/Homepage/homepage.jsx (hunk 3 above)
Comment (Megha-Joshi) [reply src/note-components/Homepage/homepage.jsx:83]: Okay

**Pull request merged**